custom_components/readme/__init__.py

```diff
@@ -185,6 +187,15 @@ def get_hacs_components(hass: HomeAssistant):
     ]
 
 
+def get_ha_installed_addons(hass: HomeAssistant) -> List[Dict[str, Any]]:
```

Comment:
Import and decorate this with `callback` from `homeassistant.core`

Reply:
done